generator/wrapper_gen.py

```diff
@@ -245,6 +251,239 @@ def daal_generate_shuffled_indices(idx, random_state):
     c_generate_shuffled_indices(data_or_file(<PyObject*>idx), data_or_file(<PyObject*>random_state))
 
 
```

Comment:
This file is so huge. Let's move the functionality with converters to another module.

Reply:
it was discussed with @Alexander-Makaryev, to keep it here like train_test split.
What is your suggestions?

Reply:
Create a file `converter_builders.py` and move this content there. From here, for example, import. When all the content is in one file, it is very difficult to read, validate and expand it.